extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/AuthRuntimeConfig.java

```diff
@@ -76,4 +76,48 @@ public interface AuthRuntimeConfig {
      * Form Auth config
      */
     FormAuthRuntimeConfig form();
+
+    /**
+     * Require that all registered HTTP authentication mechanisms must complete the authentication.
```

Comment:
```suggestion
     * Require that all registered HTTP authentication mechanisms must attempt to verify the request credentials.
```
It is an attempt to make it conditional since we now have a mode property.